tests/test_fips.py

```diff
@@ -226,37 +224,29 @@ def test_enable_fips_ppc64le_older_power(self):
 
     def test_is_fips_enabled_true(self):
         """is-fips-enabled returns 0 if fips is enabled."""
-        self.make_fake_binary('dpkg-query')
-        p = self.fips_enabled_file
-        p.write_text('1')
+        self.setup_fips(enabled=True)
         process = self.script('is-fips-enabled')
         self.assertEqual(0, process.returncode)
 
     def test_is_fips_enabled_false(self):
         """is-fips-enabled returns 1 if fips is not enabled."""
-        self.make_fake_binary('dpkg-query')
-        p = self.fips_enabled_file
-        p.write_text('0')
+        self.setup_fips(enabled=False)
         process = self.script('is-fips-enabled')
         self.assertEqual(1, process.returncode)
 
     def test_fips_cannot_be_disabled_if_enabled(self):
         """disable-fips says FIPS cannot be deactivated if it's enabled"""
-        self.make_fake_binary('dpkg-query')
-        p = self.fips_enabled_file
-        p.write_text('1')
+        self.setup_fips(enabled=True)
         process = self.script('disable-fips')
         self.assertEqual(1, process.returncode)
-        self.assertEqual(
-            'Disabling FIPS is currently not supported.',
-            process.stderr.strip())
+        self.assertIn(
+            'Disabling FIPS is currently not supported.', process.stderr)
 
     def test_disable_fips_warns_if_not_enabled(self):
```

Comment:
I think the name of this test, and its docstring, should be changed now that this condition is an error and not a warning, even though we exited non-zero before. Maybe `test_disable_fips_fails_if_not_enabled()`.